src/lib/trust_system/peer_trust.ml

```diff
@@ -196,14 +196,15 @@ module Make0 (Inputs : Input_intf) = struct
     let simple_new = Record_inst.to_peer_status new_record in
     let action_fmt, action_metadata = Action.to_log action in
     let log_trust_change () =
-      let verb =
-        if Float.(simple_new.trust > simple_old.trust) then "Increasing"
-        else "Decreasing"
-      in
-      [%log debug]
-        ~metadata:([ ("sender_id", Peer_id.to_yojson peer) ] @ action_metadata)
-        "%s trust for peer $sender_id due to %s. New trust is %f." verb
-        action_fmt simple_new.trust
+      if Float.(simple_new.trust <> simple_old.trust) then
```

Comment:
```suggestion
      if Float.(abs (simple_new.trust - simple_old.trust) < 1e-5) then
```

comparing even after some arithmetics is not sane. Unless we do know that addition is of `0`. And not `-0` for instances, because I do not recall well if `0 + (-0)` is equal to original `0`